.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,6 +13,9 @@
 - [ ] I listed at least one issue that this PR fixes in the description above.
 - [ ] I updated/added relevant documentation (doc comments with `///`).
 - [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
+- [ ] I've accounted for any new deprecations or breaking changes (Choose one).
+  - [ ] I have added a dart fix, see [Data Driven Fixes].
+  - [ ] Dart fix cannot help (should this change be reconsidered?), or this change does not need one.
```

Comment:
How about condensing this down to one check box:

```suggestion
- [ ] I followed the [breaking change policy] and added [Data Driven Fixes] where necessary.
```

I am not in love with the word "necessary" there. Maybe "appropriate" or "required" or "possible" would be better?

Reply:
Oo love that! How about `necessary` --> `supported`?

Reply:
I like "supported". 👍